pkgs/development/python-modules/merkletools/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, pysha3 }:
+
+buildPythonPackage rec {
+  pname = "merkletools";
+  version = "1.0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0pdik5sil0xcrwdcgdfy86c5qcfrz24r0gfc8m8bxa0i7h7x2v9l";
+  };
+
+  propagatedBuildInputs = [ pysha3 ];
+
+  meta = with lib; {
+    description = "Python tools for creating Merkle trees, generating Merkle proofs, and verification of Merkle proofs";
+    homepage = https://github.com/Tierion/pymerkletools;
```

Comment:
Oops, forgot about this: links should be quoted

Reply:
no, they should not be quoted

Reply:
IIRC, there's a mixture of quoted and unquoted homepage links all over nixpkgs.
Perhaps we need a mass PR to change all of these to whatever the correct usage is?

Reply:
Well, yeah, NixOs/rfcs#45 is not merged yet so the decision is not officially made yet, but I thought the consensus was to at least not to use unquoted URLs in new packages

Reply:
It's unnecessary as a mass PR will probably change it to the quoted anyway, but I'll just make it quoted in this PR.